Tema02-ToDoList/todolist.ts

```diff
@@ -0,0 +1,118 @@
+//model
+class ToDo{
+    name: string;
+    endDate: Date;
+    finished: boolean;
+    description: string;
+
+    constructor(name: string, endDate: Date, description: string){
+        this.name = name;
+        this.endDate = endDate;
+        this.finished = false;
+        this.description = description;
+    }
+
+    finsh(){
+        this.finished = true;
+    }
+
+    setName(name: string){
```

Comment:
Pentru ca vorbim totusi despre javascript, nu este cazul sa declari getter si setter. In loc de setter, poti folosi constructor, iar pentru getter, acces direct la proprietate. Doar in cazul in care mai ai logica pe langa atribuirea valorii, atunci ai putea avea o functie ca si setter :)